libssh2-sys/lib.rs

```diff
@@ -248,6 +248,30 @@ pub type LIBSSH2_PASSWD_CHANGEREQ_FUNC = extern "C" fn(
     abstrakt: *mut *mut c_void,
 );
 
+pub type LIBSSH2_USERAUTH_KBDINT_RESPONSE_FUNC = extern "C" fn(
```

Comment:
@alexcrichton here's another `libssh2-sys` change; this one exposes keyboard interactive authentication types